R/latent_model.R

```diff
@@ -168,13 +112,36 @@ epidist_gen_log_lik_latent <- function(family) {
 #' @export
 epidist_formula_model.epidist_latent_model <- function(
     data, formula, ...) {
-  # data is only used to dispatch on
+  # Update main formula
   formula <- stats::update(
     formula, delay | vreal(relative_obs_time, pwindow, swindow) ~ .
   )
```

Comment:
At one point didn't we have a formula on `pwindow` and `swindow`?

Reply:
yes see the comment in the PR about this not being scalable due to the matrix multiplication we can't avoid